configure.ac

```diff
@@ -3,4 +3,27 @@ AM_INIT_AUTOMAKE(ke-recv-extra, 0.2)
 AM_CONFIG_HEADER(config.h)
 AC_PROG_CC
 
+PKG_CHECK_MODULES(OSSO, libosso)
+AC_SUBST(OSSO_CFLAGS)
+AC_SUBST(OSSO_LIBS)
+
+PKG_CHECK_MODULES(GLIB, glib-2.0)
+AC_SUBST(GLIB_CFLAGS)
+AC_SUBST(GLIB_LIBS)
+
+PKG_CHECK_MODULES(GCONF, gconf-2.0)
+AC_SUBST(GCONF_FLAGS)
+AC_SUBST(GCONF_LIBS)
+
+PKG_CHECK_MODULES(DBUS, dbus-1)
+AC_SUBST(DBUS_CFLAGS)
+AC_SUBST(DBUS_LIBS)
+
+PKG_CHECK_MODULES(DBUS_GLIB, dbus-glib-1)
+AC_SUBST(DBUS_GLIB_CFLAGS)
+AC_SUBST(DBUS_GLIB_LIBS)
```

Comment:
All these lines can be replaced with a simple:
PKG_CHECK_MODULES([DEPS], [libosso glib-2.0 gconf-2.0 dbus-1 dbus-glib-1])

Reply:
I wouldn't necessarily lump them together. I think it's not a bad practice to separate them.

Reply:
There is no need to inflate this file.

Reply:
We have this everywhere, and I think this is the usual way to do this?

https://github.com/maemo-leste/libhildon/blob/master/configure.ac#L42
https://github.com/maemo-leste/libhildondesktop/blob/master/configure.ac#L99

etc.

Reply:
Trying to make copy-paste doesn't seem a good approach to me. We maintain pretty old code base. Much has changed in the last 10-15 years.
And we should avoid using overcomplicated things when they can be simplified.
You already could see such examples:
Old approach: https://github.com/community-ssu/ke-recv/blob/master/debian/rules
New approach: https://github.com/spinal84/ke-recv/blob/kbd-slide/debian/rules
If we want to move forward in Maemo Leste development, we should simplify things, not overcomplicate them without a need.

Reply:
I don't think there is a 'standard' way of doing this, both ways are fine IMO. Not to say that when more targets are build in a single project, using separate defines for pgkconfig flags is a must as usually includes/libs for those targets do not match. So I don't see any problem here.